Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSPatchSensitivity.py

```diff
@@ -109,8 +109,8 @@ def __patch_workspace(self, tube_in_input_ws, in_ws, patch_ws):
                 id_to_fit.append(detector_id)
             elif not inputDetInfo.isMasked(detector_idx):
                 id_to_calculate_fit.append(detector_id)
```

Comment:
@martyngigg : Not sure if I understand. Line 110?
Names are not the best: `detector_idx` is this spectrum index for this `detector_id`.
if `inputDetInfo.isMasked(detector_idx)` uses a detector ID than it's `detector_id`, otherwise it's OK.
`id_to_calculate_fit` it's just a linear scale to calculate a polynomial fitting.